### ISSUE TITLE
[Feature Request]: SQL Export - add row limit option

### ISSUE THREAD

**Issue by penCsharpener** [HeidiSQL/HeidiSQL#341]
## Use case

This is related to issue 340 as well but not worth linking with #. 
When exporting data from a production to a local/sandbox instance of MySQL a developer doesn't always need all the rows of a large table. That just adds unnecessary traffic and time.

## Current behavior

Export entire table or nothing or just table structure.

## Expected behavior

An option that allows the developer to just export / copy the last 100 or 1000 rows.

## Possible solution

The menu would need a column dropdown for the table in question by which the sorting happens, a sort order (ASC or DESC) and the number of rows. Heidi could prefilter to only include id columns or date columns. I don't see any use for text fields. 

## Environment

* **HeidiSQL version:** 
  9.5.0.5293
* **Database system and version:** 
  MySQL 5.7.23 and MariaDB 10.3.6
* **Operating system:** 
  Win 10 1803

**Comment by ansgarbecker** [comment]
A sorting order (and limit also) would be on a per-table basis. So that could live in the left tree in a new column with a dropdown when editing.
![grafik](https://user-images.githubusercontent.com/7986591/46674677-124c4780-cbdd-11e8-9ed9-9488f94e1c22.png)

**Comment by penCsharpener** [comment]
yes, good idea. But in your mockup you have a "name". I don't quiet see a use for alphabetical sorting.

**Comment by ansgarbecker** [comment]
I can already see others asking "where are the other column names?" :)

**Comment by Abadii** [comment]
This one is really required! Hope to see it in the new build

**Comment by gjain2** [comment]
Could you please confirm if this limit option or something similar was added in Heidi or not? I can't find any option to limit record but i don't want to export all data as few tables have a lot of records.

**Comment by ansgarbecker** [comment]
Since #2168 HeidiSQL auto-sorts data by the table's primary key if there is one. That already solved integrity problems on PostgreSQL.

What is missing for a data limitation/filtering is more precisely defined in #1674 . I'm aware that a WHERE filter is not a sorting option, but in conjunction with the existing auto-sorting that WHERE filter option seems most reasonable to me, and would solve this feature best.
